gazebo_ros/scripts/spawn_model

```diff
@@ -288,6 +288,7 @@ if __name__ == "__main__":
         print usage()
     else:
         print("spawn_model script started") # make this a print incase roscore has not been started
+        rospy.init_node('spawn_model')
```

Comment:
If you left `rospy.init_node()` inside of `SpawnModel.callSpawnService()`, then it would not get called before `SpawnModel.__init__()` and then would not get called before `rospy.get_param()`.